src/utils/fetchClient.ts

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
-const BASE_URL = 'https://mate.academy/students-api';
+const BASE_URL = 'https://mate.academy/students-apik';
```

Comment:
GET https://mate.academy/students-apik/todos?userId=10914 net::ERR_FAILED 404 I guess It's not correct behavior You should check your BASE_URL